pkg/http/api.go

```diff
@@ -249,6 +250,17 @@ func (a *api) constructMetadataEndpoints() []Endpoint {
 	}
 }
 
+func (a *api) constructHealthzEndpoints() []Endpoint {
+	return []Endpoint{
+		{
+			Methods: []string{http.Get},
+			Route:   "healthz",
```

Comment:
is it the same endpoint, how would liveness be distinguished from readiness?

Reply:
> is it the same endpoint, how would liveness be distinguished from readiness?

Currently they are distinguishable by different probe parameters. I can have separate endpoints but decided to have same endpoint with different probe parameters. Though it is open for discussion.

Reply:
one endpoint is fine, what are the different params? do you mean query params?

Reply:
No, I meant Probe configuration paratmeters : 

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes

Reply:
Recommended to keep as one endpoint for both.

Reply:
The previous product(AppCenter) used single endpoint(/status) for both purposes (liveness/readiness) in k8s cluster because we was not able to find the difference between readiness and liveness. I think the same thing can be applied to dapr. What's the difference between readiness and liveness in the view of Dapr ? if we cannot find it, then we can use single one for both purposes.

Reply:
Actually, Dapr does distinct between the two, but those could still be served by one endpoint, provided we make a small change.

Dapr is considered `ready` when init has finished, not when the API is ready to serve requests.
We need to guarantee that the endpoint will return `200OK` when when all the init steps are done.

And it is OK to use the same endpoint for liveness and readiness in that case.